ratis-logservice/src/test/resources/logservice.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0"?>
+<?xml-stylesheet type="text/xsl" href="configuration.xsl"?>
+<!--
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+-->
+<configuration>
+  <property>
+    <name>logservice.archival.location</name>
+    <value>target/tmp/archive_1/</value>
```

Comment:
Ah! Tricky.

Nice to have this set here -- should we have individual tests overriding this so we can have each test keep their data around for debugging purposes?

Reply:
Let me see if I can do that, not sure if we need to restart the cluster if we change the config in the same test class.

Reply:
I tested archival through some exports by providing different locations.